package.json

```diff
@@ -4,28 +4,27 @@
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": ">= 16.4.0"
+    "node": "^20.19.0"
   },
   "scripts": {
     "clean": "rimraf dist && lerna exec -- rimraf dist tsconfig.tsbuildinfo",
-    "build": "tsc -b packages",
+    "build": "tsc -b packages && ts-node tools/test-dist",
     "build:watch": "yarn build --watch",
-    "postbuild": "ts-node tools/test-dist",
-    "docs": "yarn build && npx typedoc",
+    "docs": "yarn build && yarn dlx typedoc",
     "lerna:publish": "lerna publish --force-publish --conventional-commits --no-changelog --exact",
     "lint:js": "prettier --check . --experimental-cli && eslint . --cache",
     "lint:markdown": "electron-markdownlint \"**/*.md\"",
     "lint:markdown-js": "electron-lint-markdown-standard --root . --ignore-path .markdownlintignore --semi \"**/*.md\"",
     "lint:markdown-links": "electron-lint-markdown-links --root . --ignore-path .markdownlintignore \"**/*.md\"",
     "lint": "npm run lint:js && npm run lint:markdown && npm run lint:markdown-js && npm run lint:markdown-links",
     "lint:fix": "prettier --write . --experimental-cli && eslint --fix . --cache",
-    "link:prepare": "lerna exec -- node ../../../tools/silent.js yarn link --silent --no-bin-links --link-folder ../../../.links",
-    "link:remove": "lerna exec -- node ../../../tools/silent.js yarn unlink --silent --no-bin-links --link-folder ../../../.links",
+    "link:prepare": "lerna exec -- node ../../../tools/silent.js yarn link --silent --link-folder ../../../.links",
```

Comment:
Does the `--link-folder` flag still work? I don't see it documented in https://yarnpkg.com/cli/link

Reply:
I'm gonna go ahead and remove the entire script, I don't think we need the `link:prepare` and `link:remove` with yarn v4 workspaces